spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 # Must be explicit as activerecord is optional dependency
 require 'active_record/railtie'
 
+unless File.exist?('config/database.yml')
```

Comment:
because config/environment is throwing the error we might have to do the following
```
begin
  require File.expand_path('../../config/environment', __FILE__)
rescue TypeError => e
  fail 'RSPEC currently needs a configured database'
end
```